### ISSUE TITLE
Try again counting package use on GitHub

### ISSUE THREAD

**Issue by jeroen** [r-universe-org/help#380]
Github code search seems to have gotten better.

Maybe we can use the search to find scripts with `library(pkg)` as a measure of end-user usage. This may help boost the rank for packages that are only used by end-users, but not much depended on by other packages.

For example: https://github.com/search?q=%22library%28jsonlite%29%22+path%3A*.R&type=code&query=%22library%28jsonlite%29%22+path%3A*.R

Also the goal of this measure is not to be exact, but proxy measure of end-user usage. So I think only looking at `library()` is probably enough.

**Comment by jeroen** [comment]
Sadly it seems this is not exposed via the GH API. 

Looking at chrome devtools it seems to be calling a special "blackbird_count" api which is protected

https://github.com/search/blackbird_count?saved_searches=&q=%22library%28ggplot2%29%22+path%3A*.R

**Comment by jeroen** [comment]
Added this by scraping, hopefully it will become part of the public API.

**Comment by bastistician** [comment]
I noticed that the current "scripts" count misses `library("jsonlite")` calls, which use a character argument instead of the NSE variant.

According to a `grep` over my local projects I seem to prefer the NSE variant (which is also used in the base R manuals) but the quoted variant isn't rare. Certainly a matter of taste.

Current counts for `jsonlite`:

- <https://github.com/search?q=library%28jsonlite%29&type=code>: 24.8k files
- <https://github.com/search?q=library%28%22jsonlite%22%29&type=code>: 1.4k files
- <https://github.com/search?q=library%28%27jsonlite%27%29&type=code>: 426 files

For comparison: `grep`ing in `demo/*.{R,r}` scripts in CRAN packages currently gives 328 packages (81%) using NSE, 65 packages (16%) using double quotes, and 13 packages (3%) using single quotes.

**Comment by jeroen** [comment]
Yes I am aware of that. There are many ways to load a package, including `require("pkg")` and `pkg::` etc. We look only at the most common form `library(pkg)` as a ___proxy___ for how often a package is used in scripts.  This metric is not intended to be an exact score, but rather just one measure that enables us to compare it to other packages.

So for example we see 28k hits for `library(jsonlite)` and only 6k for `library(rjson)` and 1.6k hits for `library(RJSONIO)` which gives us an indication of the relative popularity of each package.
